src/main/java/bowling/domain/NormalFrame.java

```diff
@@ -32,4 +35,40 @@ private Frame getFrame() {
         next = new NormalFrame(frameNumber + 1);
         return next;
     }
+
+    public boolean hasNext() {
+        return Objects.nonNull(next);
+    }
+
+
+    @Override
+    public Score getScore() {
+        if (pitch.getState() == State.GUTTER || pitch.getState() == State.NORMAL || pitch.getState() == State.MISS) {
+            return pitch.getScore();
+        }
+
+        if (!hasNext()) {
+            return Score.of(UN_SCORE, 0);
```

Comment:
```suggestion
            return Score.unScore();
```

UnScore또한 Score로 넘길 수 있을 것 같아요 ㅎㅎ